sanic_openapi/spec.py

```diff
@@ -5,6 +5,7 @@ class Spec:
     swagger = "2.0"
 
     def __init__(self, app: Sanic) -> None:
+        self.swagger = self.__class__.swagger
```

Comment:
What is the purpose of this line?

Reply:
@chenjr0719 From the previous version sanic-openapi variable _spec must contain 'swagger' key (line 72 in swagger.py ver 0.6.0):
```
@swagger_blueprint.listener("before_server_start")
def build_spec(app, loop):
    _spec["swagger"] = "2.0"
    _spec["info"] = {

```

Without it  Spec().as_dict won't have 'swagger' key.

Reply:
I think this is because we put it as a class variable in the `Spec` class but when use `_spec.__dict__`, it only returns the instance variables.

How about we remove the class variable and assign it in `__init__()`? I think I also need some feedback from @AtomsForPeace.

Reply:
Yeah, I can't think of a nicer way of doing it really.

Reply:
Hello! I rewrited the 'swagger' variable as an instance variable.
thanks